src/components/forms/GeneralForm.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <section class="general-form">
+    <form @submit.prevent="generateFurniture">
+      <div>
+        <label>Altura en cm?</label>
+        <input type="number" v-model="$store.state.furniture.totalHeight" />
+      </div>
```

Comment:
v-model to create a local instance of furniture and when submit, save it in store